lib/content/getting-started.md

```diff
@@ -248,7 +238,7 @@ Moving on!
 
 Immediately, we see that our strategy of hardcoding our riddles within our first route's handler is, although expedient, unworkable. Our other routes will need to know about that data (let alone any other pieces of our riddle-sharing application we build later). So, let's centralize it.
 
-There's a whole slew of ways you handle this. A simple one: we just create a file called `data` under `lib` and set it up to export our riddles.
+Though there's a whole slew of ways you handle this, we'll create a file called `data` under `lib` and set it up to export our riddles.
```

Comment:
I wonder if it might be a little more comfy if we don't mention that there are tons of ways to solve the problem, and just go ahead with a solution that will work for us.
```suggestion
For our purposes, we'll create a file called `data` under `lib` and set it up to export our riddles.
```

Reply:
Ay, yea, that reads much better. Thanks!